course/src/main/java/org/openedx/course/presentation/container/CourseHomeTab.kt

```diff
@@ -0,0 +1,20 @@
+package org.openedx.course.presentation.container
+
+import androidx.annotation.StringRes
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.Chat
+import androidx.compose.material.icons.automirrored.filled.TextSnippet
+import androidx.compose.material.icons.filled.Home
+import androidx.compose.material.icons.outlined.CalendarMonth
+import androidx.compose.material.icons.rounded.PlayCircleFilled
+import androidx.compose.ui.graphics.vector.ImageVector
+import org.openedx.core.ui.TabItem
+import org.openedx.course.R
+
+enum class CourseHomeTab(@StringRes override val title: Int, override val icon: ImageVector) : TabItem {
```

Comment:
Let's rename it to `CourseContainerTab` because CourseHome is itself a tab. Thoughts?